### ISSUE TITLE
Incomplete "Move user profiles" option in Windows 10

### ISSUE THREAD

**Issue by GWeck** [QubesOS/qubes-issues#7413]
### Qubes OS release

Qubes R4.1 with Qubes Windows Tools 4.1-67.1

### Brief summary
If during QWT installation, the option `Move user profiles` is selected for a Windows 10 TemplateVM, the hidden directory AppData is not moved from drive `C:` to `Q:`. This has the consequence that any application configuration data stay in the template. Changes of such configuration data in an AppVM will be lost on AppVM shutdown.

### Steps to reproduce
Install QWT 4.1.67.1 with the option `Move user profiles`.

### Expected behavior
All user data including the folder `AppData` will be moved from drive `C:`to drive `Q:`.

### Actual behavior
Only the folder `documents`  will be moved. The `AppData` folder stays on drive `C:`, causing application configuration changes to be lost on AppVM shutdown. This is probably caused by using the Windows `copy` operation which seems to forget hidden folders. (For Windows 7, the function works correctly.)

**Comment by jevank** [comment]
I cannot reproduce it with Windows 10. `Users` folder completely moved to Q:

**Comment by jevank** [comment]
Perhaps this is specific to application issue. Could you check with manually creating files and folders in  `Appdata`?

**Comment by GWeck** [comment]
This was caused by leftover data from a Qubes R4.0.4/QWT 4.0.1.3 installation. If all private data are kept on drive `C:`, `Move user profiles` works as expected and moves everything to drive `Q:`.

Also, renaming an old private disk `D:` to `Q:` for Windows 7 is only necessary if such a drive was left over from an earlier version of Qubes/QWT.

I updated the installation documentation accordingly.

Sorry for the inconvenience!
